packages/@aws-cdk/aws-lambda/test/test.vpc-lambda.ts

```diff
@@ -43,6 +43,81 @@ export = {
 
       test.done();
     }
+	
+	 public 'has securitygroup that is passed in props'(test: Test) {
```

Comment:
Check your indents here as well. Keep it consistent with the existing content in this file.

Reply:
Something very weird here. Even after fixing this one seems to reappear. Will fix in a while.